libraries/AP_Camera/examples/send_camera_information.py

```diff
@@ -0,0 +1,152 @@
+        self.connection.wait_heartbeat()
+        print("Heartbeat received from system (system %u component %u)" % (self.connection.target_system, self.connection.target_component))
+        self.connection.mav.heartbeat_send(
+            type=mavutil.mavlink.MAV_TYPE_ONBOARD_CONTROLLER,
+            autopilot=mavutil.mavlink.MAV_AUTOPILOT_INVALID,
+            base_mode=0,
+            custom_mode=0,
+            system_status=mavutil.mavlink.MAV_STATE_UNINIT,
+            mavlink_version=3
+        )
+
+    def send_camera_information(self):
+        flags = (
+            CAMERA_CAP_FLAGS_CAPTURE_VIDEO |
+            CAMERA_CAP_FLAGS_CAPTURE_IMAGE |
+            CAMERA_CAP_FLAGS_HAS_MODES |
+            CAMERA_CAP_FLAGS_CAN_CAPTURE_IMAGE_IN_VIDEO_MODE |
+            CAMERA_CAP_FLAGS_CAN_CAPTURE_VIDEO_IN_IMAGE_MODE |
+            CAMERA_CAP_FLAGS_HAS_IMAGE_SURVEY_MODE |
+            CAMERA_CAP_FLAGS_HAS_BASIC_ZOOM |
+            CAMERA_CAP_FLAGS_HAS_BASIC_FOCUS |
+            CAMERA_CAP_FLAGS_HAS_VIDEO_STREAM |
+            CAMERA_CAP_FLAGS_HAS_TRACKING_POINT |
+            CAMERA_CAP_FLAGS_HAS_TRACKING_RECTANGLE |
+            CAMERA_CAP_FLAGS_HAS_TRACKING_GEO_STATUS
+        )
+        self.connection.mav.camera_information_send(
+            int(time.time() * 1000) & 0xFFFFFFFF,          # time_boot_ms
+            b"CameraVendor123" + b'\0' * (32 - len("CameraVendor123")),  # vendor_name
+            b"CameraModel123" + b'\0' * (32 - len("CameraModel123")),    # model_name
+            (1 << 24) | (0 << 16) | (0 << 8) | 1,          # firmware_version
+            float('nan'),                                  # focal_length
+            float('nan'),                                  # sensor_size_h
+            float('nan'),                                  # sensor_size_v
+            self.resh,                                     # resolution_h
+            self.resv,                                     # resolution_v
+            0,                                             # lens_id
+            flags,                                         # flags
+            0,                                             # cam_definition_version
+            b"",                                           # cam_definition_uri
+            0                                              # gimbal_device_id
+        )
+        print("Camera information message sent")
+
+    def handle_camera_track_point(self, msg):
+        print("Received MAV_CMD_CAMERA_TRACK_POINT command.")
+        # These are already floats
+        param1 = msg.param1
+        param2 = msg.param2
+        print(f"Tracking point parameters: param1={param1}, param2={param2}")
+
+    def handle_camera_track_rectangle(self, msg):
+        print("Received MAV_CMD_CAMERA_TRACK_RECTANGLE command.")
+        # These should remain as floats (normalized coordinates)
+        norm_x = msg.param1
+        norm_y = msg.param2
+        norm_w = msg.param3
+        norm_h = msg.param4
```

Comment:
https://mavlink.io/en/messages/common.html#MAV_CMD_CAMERA_TRACK_RECTANGLE

- param3 is bottom right corner x not width
- param4 is bottom right corner y not height